tests/test_phase_psf.py

```diff
@@ -19,7 +19,7 @@
 from __future__ import print_function
 import os
 import numpy as np
-from galsim_test_helpers import *
+from galsim_test_helpers import timer, do_shoot, do_pickle, all_obj_diff, getmoments
```

Comment:
Yeah, I'm pretty ambivalent to this too, but pyflakes isn't.  So importing everything by name here is just to make it easier to verify that pyflakes isn't suggesting anything legitimate.